### DISCUSSION TITLE
BIGTREETECH SKR V1.4 (LPC1768 & LPC1769 Turbo) & GTR V1.0 (STM32F407)

### DISCUSSION THREAD

**Discussion by K2AAE** [terjeio/grblHAL#179] [Ideas]
I was looking into updating an engraver I have but I can't find anything for BIGTREETECH SKR V1.4 (LPC1768 & LPC1769 Turbo) & GTR V1.0 (STM32F407) - I found a post on LPC1768 where you talked about adding SKR V1.3 (LPC1768) support but after downloading GIT I found nothing in the drivers about it. Note: The SKR E3 Turbo uses LPC1769 with TMC2209 UART soldered on board and the E3 Mini has STM32F103 with TMC2209 UART.
Thanks
FYI: I noticed STM's new toy STM32H735G-DK 550Mhz 1Mbyte Flash 4.3" Touch screen --  Mated with CNC Breakout board for Nucleo-64?

**Comment by terjeio** [comment]
Check out the [test branch](https://github.com/terjeio/grblHAL/tree/test).

> FYI: I noticed STM's new toy STM32H735G-DK 550Mhz 1Mbyte Flash 4.3" Touch screen -- Mated with CNC Breakout board for Nucleo-64?

There are many possibilities to expand grblHAL - you want to write a driver for this?

**Reply by K2AAE** [reply]
My project cue overflowed the year I got involved with STM32MP157A-DK1 and then the Nvidia Jetson Nano/ Xavier NX. I'll have to look at the STM32H735G-DK just to get back to a compile cycle that doesn't require you to learn how to 'bake'.
